sdk/storage/src/account/account/requests/get_account_information_builder.rs

```diff
@@ -1,54 +1,47 @@
 use crate::account::account::responses::GetAccountInformationResponse;
 use crate::core::prelude::*;
-use azure_core::errors::AzureError;
-use hyper::{Method, StatusCode};
 
 #[derive(Debug, Clone)]
-pub struct GetAccountInformationBuilder<'a, C>
-where
-    C: Client,
-{
-    client: &'a C,
+pub struct GetAccountInformationBuilder<'a> {
+    storage_client: &'a StorageClient,
 }
 
-impl<'a, C> GetAccountInformationBuilder<'a, C>
-where
-    C: Client,
-{
-    #[inline]
-    pub(crate) fn new(client: &'a C) -> GetAccountInformationBuilder<'a, C> {
-        GetAccountInformationBuilder { client }
+impl<'a> GetAccountInformationBuilder<'a> {
+    pub(crate) fn new(storage_client: &'a StorageClient) -> Self {
+        Self { storage_client }
     }
 }
 
-impl<'a, C> ClientRequired<'a, C> for GetAccountInformationBuilder<'a, C>
-where
-    C: Client,
-{
-    #[inline]
-    fn client(&self) -> &'a C {
-        self.client
-    }
-}
+impl<'a> GetAccountInformationBuilder<'a> {
+    pub async fn execute(
+        self,
+    ) -> Result<GetAccountInformationResponse, Box<dyn std::error::Error + Send + Sync>> {
+        let mut url = self
+            .storage_client
+            .storage_account_client()
+            .blob_storage_url()
+            .to_owned();
+
+        url.query_pairs_mut().append_pair("restype", "account");
+        url.query_pairs_mut().append_pair("comp", "properties");
 
-// methods callable only when every mandatory field has been filled
-impl<'a, C> GetAccountInformationBuilder<'a, C>
-where
-    C: Client,
-{
-    #[inline]
-    pub async fn finalize(self) -> Result<GetAccountInformationResponse, AzureError> {
-        let uri = format!(
-            "{}/?restype=account&comp=properties",
-            self.client.blob_uri()
-        );
-        trace!("uri == {:?}", uri);
-
-        let (headers, _) = self
-            .client()
-            .perform_request(&uri, &Method::GET, &|request| request, None)?
-            .check_status_extract_headers_and_body(StatusCode::OK)
+        trace!("url == {:?}", url);
```

Comment:
Worth a TODO to remove or change this later. We'll want a generate tracing solution (or maybe we can use this still as the entry point if we can override the implementation), but this should include call info, for example. We do this consistently across languages for the SDK.